### PR TITLE
Adding ability to specify rules via the command line

### PR DESCRIPTION
**Pull request by src-code**
```
$ atomizer 

  Usage: atomizer [options] [path]

  Options:

    -h, --help                          output usage information
    -V, --version                       output the version number
    -R, --recursive                     process all files recursively in the path
    -c, --config [file]                 source config file
    -r, --rules [file]                  custom rules file
    -o, --outfile [file]                destination config file
    -n, --namespace [namespace]         adds the given namespace to all generated Atomic CSS selectors
    -H, --helpersNamespace [namespace]  adds the given namespace to all helper selectors
    --rtl                               swaps `start` and `end` keyword replacements with `right` and `left`
    --ie                                adds old IE hacks to the output
    --verbose                           show additional log info (warnings)
```

Can specify multiple rule files by specifying multiple `-r` params:

```
atomizer -r ./rules1.js -r ./rules2.js my_file.html
```

FYI @renatoi

### CODE REVIEW

**Comment by src-code** [comment]
fyi @dnukem

File: bin/atomizer [hunk 1]
```diff
@@ -20,17 +20,23 @@ var content = '';
...
-  .option('-c, --config [file]', 'source config file if any.')
-  .option('-o, --outfile [file]', 'destination config file.')
-  .option('-n, --namespace [namespace]', 'adds the given namespace to all generated Atomic CSS selectors.')
-  .option('-H, --helpersNamespace [namespace]', 'adds the given namespace to all helper selectors.')
-  .option('--rtl', 'swaps `start` and `end` keyword replacements with `right` and `left`.')
-  .option('--ie', 'adds old IE hacks to the output.')
-  .option('--verbose', 'show additional log info (warnings).')
+  .option('-R, --recursive', 'process all files recursively in the path')
+  .option('-c, --config [file]', 'source config file')
+  .option('-r, --rules [file]', 'custom rules file', collect, [])
```
Comment (dnewcome) [inline bin/atomizer]: Help text should hint that multiple files may be specified I think

Comment (src-code) [inline bin/atomizer]: Done!

**Comment by renatoi** [comment]
:+1: I'd also update the README.md reference.

**Comment by src-code** [comment]
@renatoi Good catch!  Forgot about the README

**Pull request merged**